docs/operations.md

```diff
@@ -0,0 +1,64 @@
+| title                  |
+|------------------------|
+| BTP Manager Operations |
+
+## Overview
+
+BTP Manager performs the following operations:
+- provisioning of SAP BTP Service Operator
+- deprovisioning of SAP BTP Service Operator and its resources, Service Instances and Service Bindings
+
+## Provisioning
+
+### Prerequisites
+
+The prerequisites for SAP BTP Service Operator provisioning are:
+- Namespace `kyma-system`
+- PriorityClass `kyma-system`
+- Secret `sap-btp-manager` with data for SAP BTP Service Operator
+
+Namespace and PriorityClass resources are created during Kyma installation. Secret is injected into the cluster by Kyma Environment Broker. If you want to provision SAP BTP Service Operator on a cluster without Kyma then you must create prerequisites yourself.
+
+### Process
+
+The provisioning process is a part of a module reconciliation and is carried out accordingly to the diagram presented below:
+
+![Provisioning diagram](./assets/provisioning.svg)
```

Comment:
hard to read text in the condition objects, adjust that text resides inside the component

Reply:
Second things, diagram for provisioning and de-provisioning , all elements, text must look the same. As well if not added, include sources from draw.io into btp-manager repository.

Reply:
suggestion applied, I modified deprovisioning diagram to match the provisioning one.